### DISCUSSION TITLE
Bewegliches Hud nicht da

### DISCUSSION THREAD

**Discussion by goose84** [Courseplay/Courseplay_FS22#1786] [Q&A] [answered]
Hallo, bei mir lässt sich der Hud nicht mehr öffnen.
Finde ich keine Tastenbelegung wie früher dazu. 
![fsScreen_2022_07_20_13_05_34](https://user-images.githubusercontent.com/109667014/179970713-bea9f5fa-9bcb-4ec9-b547-0951749f3916.png)
![fsScreen_2022_07_20_13_25_59](https://user-images.githubusercontent.com/109667014/179970718-14ce6bde-7a5a-4712-97ce-19299829615f.png)

**Comment by Tensuko** [comment, marked as answer]
Du hast das Gamepad Freundliche HUD aktiviert -> Globale Einstellungen.

**Comment by PhilippS1** [comment]
Hallo zusammen,
bei mir ist wärend des spielens der HUD verschwunden. Ich vermute das Fenster hat sich ausversehen an den Rand verschoben. Das Helfer Infofenster kann ich mit ganz viel Geduld soweit in den Rand schieben das ich es auch sogut wie nicht mehr sehen kann. Auch in jedem anderen Savergame ist der HUD verborgen, selbst wenn ich ein neues Savegame anlege. Gibt es eine Möglichkeit mein Problem zu fixen ohne das ich alle meine Kurse verliere?
Gruß

**Reply by Tensuko** [reply]
Unter "..\Dokumente\My Games\FarmingSimulator2022\modSettings\FS22_Courseplay" befindet sich die courseplay.xml
Da drin gibt es die Zeilen:
```
<Hud posX="0.205860" posY="0.018750"/>
    <HudInfoTexts posX="0.706901" posY="0.881713"/>
```
Die einmal löschen, dann sollten sie wieder da sein.
Oder du benutzt die Zeilen die ich dir hier rein geschrieben habe um deine Zeile zu ersetzen.

**Comment by PhilippS1** [comment]
Hat funktioniert, danke schön
